working_agreement.md

```diff
@@ -28,14 +29,14 @@
 1. Building a diverse environment
     - Please be open to others perspectives, engage with them, and be willing to revise your own
     - We work to vocally support diverse perspectives
-    - To specifically amplify the work of Black, Indigenous, People-of-Color (BIPOC) researchers, we've committed to 
-      - Discussing an article written by a BIPOC author at least once a semester during lab meetings. Everyone is encouraged to lead a discussion
```

Comment:
Engaging in a lab discussion regarding systemic problems within the field of ecology and evolutionary biology or within graduate school (i.e., decolonizing conservation, systemic barriers and racism within the field of ecology, history of racism against BIPOC communities in genomics)